recipes/minimize_access.rb

```diff
@@ -53,3 +53,13 @@
   mode '0750'
   not_if { node['os-hardening']['security']['users']['allow'].include?('change_user') }
 end
+
+# /var/log should restricted to root or syslog on ubuntu > 14.04
+directory '/var/log' do
+  owner 'root'
+  if node['platform'] == 'ubuntu'
+    group Chef::VersionConstraint.new('>= 14.04').include?(node['platform_version']) ? 'syslog' : 'root'
```

Comment:
@shoekstra do we need the version constraint? Is the check for `ubuntu`is not enough here?

Reply:
@artem-sidorenko just checked and 12.04 LTS is EOL, so it can be removed.